README.md

```diff
@@ -27,16 +27,14 @@ _From [Mux](https://mux.com?utm_source=github&utm_medium=social&utm_campaign=med
 
 Visit the [official documentation for Media Chrome](https://media-chrome.org/en/get-started) for the latest up-to-date usage instructions.
 
-You can find the [current release candidate documentation here.](https://v1.media-chrome.org/en/get-started)
```

Comment:
should we invert this? And then in a month or two, we can remove this.

```
You can find the [documentation for v0.x of media-chrome here.](https://v0.media-chrome.org/en/get-started)
```

Reply:
makes sense, will edit